jinahub/indexers/dbms/PostgreSQLDBMSIndexer/postgreshandler.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         **kwargs,
     ):
         super().__init__(*args, **kwargs)
-        self.logger = JinaLogger(self.__class__.__name__)
+        self.logger = JinaLogger('psq_handler')
```

Comment:
Shouldn't this be `get_logger(self)` as well?

Reply:
This is not an executor, it's the PSQL conn handler

Reply:
Fair enough.